### PR TITLE
add note to cloud sso faq

### PR DESCRIPTION
**Pull request by MichaelTrestman**

### CODE REVIEW

File: cockroachcloud/cloud-sso.md [hunk 1]
```diff
@@ -73,7 +73,9 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
 
 ### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
 
-Enterprise Authentication supports only a flow initiated by {{ site.data.products.db }}. Initiation from the IdP is not supported.
+The primary flow is to initiate {{ site.data.products.db }} SSO through the cloud console UI.
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:76]:
```suggestion
The primary flow is to initiate {{ site.data.products.db }} SSO through the Cockroach Cloud Console.
```

File: cockroachcloud/cloud-sso.md [hunk 2]
```diff
@@ -73,7 +73,9 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
...
+
+Initiation from the IdP is also possible, but not supported through the cloud console. Contact your account team for more information.
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:78]: I wonder if it makes sense to rephrase this as "If you need to initiate SSO integration from the IdP, contact your account team for assistance."

Comment (MichaelTrestman) [inline cockroachcloud/cloud-sso.md:78]:
```suggestion
If you need to initiate SSO integration from the IdP, contact your account team for assistance
```

File: cockroachcloud/cloud-sso.md [hunk 3]
```diff
@@ -73,9 +73,9 @@ Yes, as long as Enterprise Authentication is enabled for your {{ site.data.produ
 
 ### Which authentication flows are supported with Enterprise Authentication? Is it possible to enable the identity provider initiated flow?
 
-The primary flow is to initiate {{ site.data.products.db }} SSO through the cloud console UI.
+The primary flow is to initiate {{ site.data.products.db }} SSO through the Cockroach Cloud Console.
 
-Initiation from the IdP is also possible, but not supported through the cloud console. Contact your account team for more information.
+If you need to initiate SSO integration from the IdP, contact your account team for assistance
```
Comment (mdlinville) [inline cockroachcloud/cloud-sso.md:78]:
```suggestion
If you need to initiate SSO integration from the IdP, contact your account team for assistance.
```

**Review by mdlinville — APPROVED** [review]
